LayoutTests/accessibility/shadow-dom/reference-target/mac/aria-controls.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE HTML>
+<html>
+
```

Comment:
A few extra newlines in between tags here too. I don't think there's a hard rule here, but in many other accessibility tests we only have empty newlines between "significant" sections of elements in the `<body>` (e.g. I'd say the ones between `<body>` and `<x-panel-2>` are good here).